twake/backend/node/src/services/channels/services/channel/activities/service.ts

```diff
@@ -0,0 +1,155 @@
+import { logger } from "../../../../../core/platform/framework";
+import { localEventBus } from "../../../../../core/platform/framework/pubsub";
+import { Channel as ChannelEntity } from "../../../entities/";
+import { ResourceEventsPayload } from "../../../../types";
+import { GenericObjectType } from "./types";
+import _ from "lodash";
+
+export default class Activities {
+  version = "1";
+
+  init(): Activities {
+    const channelMemberCreatedEvent = "channel:member:created";
```

Comment:
Put all your constants like this on the top of the file

Reply:
Done